<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScanInputDlgUI</class>
 <widget class="QDialog" name="ScanInputDlgUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>575</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Lucida Grande</family>
    <pointsize>9</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
    <underline>false</underline>
    <strikeout>false</strikeout>
   </font>
  </property>
  <property name="windowTitle">
   <string>Scan command generator</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="maximumSize">
      <size>
       <width>32767</width>
       <height>32767</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Lucida Grande</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Single device scan</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="0">
        <widget class="QGroupBox" name="groupBox10">
         <property name="title">
          <string>Scan parameters</string>
         </property>
         <layout class="QGridLayout">
          <property name="margin">
           <number>9</number>
          </property>
          <property name="spacing">
           <number>6</number>
          </property>
          <item row="3" column="0">
           <widget class="QLabel" name="tL_sscan_Preset">
            <property name="text">
             <string>&lt;b&gt;Preset:&lt;/b&gt;</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLabel" name="tL_sscan_PresetS">
            <property name="text">
             <string>sec</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="scanPreset"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="tL_sscan_Start">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Start/Center:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="tL_sscan_Step">
            <property name="text">
             <string>&lt;b&gt;Step size:&lt;/b&gt;</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="tL_sscan_NStep">
            <property name="text">
             <string>&lt;b&gt;Number of steps:&lt;/b&gt;</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="2">
           <widget class="QLineEdit" name="scanNumsteps"/>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QLineEdit" name="scanStep"/>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QLineEdit" name="scanStart"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QGroupBox" name="scanType">
         <property name="title">
          <string>Scan type</string>
         </property>
         <layout class="QGridLayout" name="_2">
          <property name="margin">
           <number>9</number>
          </property>
          <property name="spacing">
           <number>6</number>
          </property>
          <item row="0" column="0">
           <widget class="QRadioButton" name="scanSingle">
            <property name="text">
             <string>single-sided</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QRadioButton" name="scanCentered">
            <property name="text">
             <string>centered</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="1">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>50</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="tL_sscan_device">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Select device:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignTop</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QListWidget" name="deviceList">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="tL_sscan_device_2">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;or enter a device name:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="deviceName"/>
         </item>
        </layout>
       </item>
       <item row="2" column="1">
        <widget class="QGroupBox" name="groupBox11">
         <property name="title">
          <string>Calculated Values</string>
         </property>
         <layout class="QGridLayout">
          <property name="margin">
           <number>9</number>
          </property>
          <property name="spacing">
           <number>6</number>
          </property>
          <item row="1" column="0" colspan="3">
           <widget class="QPushButton" name="scanCalc">
            <property name="text">
             <string>Calculate</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="tL_sscan_End_2">
            <property name="text">
             <string>&lt;b&gt;Move time per step:&lt;/b&gt;</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLineEdit" name="scanMovetime"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="tL_sscan_End">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Range:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="2">
           <widget class="QLineEdit" name="scanRange"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="tL_sscan_PresetH">
            <property name="text">
             <string>&lt;b&gt;Est. time:&lt;/b&gt;</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1" colspan="2">
           <widget class="QLineEdit" name="scanEstimation"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="TabPage">
      <attribute name="title">
       <string>Q-space scan</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="qscanType">
         <property name="title">
          <string>Scan type</string>
         </property>
         <layout class="QGridLayout">
          <property name="margin">
           <number>9</number>
          </property>
          <property name="spacing">
           <number>6</number>
          </property>
          <item row="1" column="1">
           <widget class="QRadioButton" name="qscanTrans">
            <property name="text">
             <string>transversal</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QRadioButton" name="qscanCentered">
            <property name="text">
             <string>centered</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QRadioButton" name="qscanSingle">
            <property name="text">
             <string>single-sided</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QRadioButton" name="qscanRandom">
            <property name="text">
             <string>random direction</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="qscanLong">
            <property name="text">
             <string>longitudinal</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox3">
         <property name="title">
          <string>Start and Steps</string>
         </property>
         <layout class="QVBoxLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="margin">
           <number>9</number>
          </property>
          <item>
           <layout class="QHBoxLayout">
            <property name="spacing">
             <number>15</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <layout class="QGridLayout">
              <property name="margin">
               <number>0</number>
              </property>
              <property name="spacing">
               <number>15</number>
              </property>
              <item row="2" column="0">
               <widget class="QLabel" name="textLabel10_3">
                <property name="text">
                 <string>&lt;b&gt;l&lt;/b&gt;</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="kInput">
                <property name="text">
                 <string>0.0</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLineEdit" name="EInput">
                <property name="text">
                 <string>0.0</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="lInput">
                <property name="text">
                 <string>0.0</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="hInput">
                <property name="text">
                 <string>0.0</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="textLabel10_2">
                <property name="text">
                 <string>&lt;b&gt;k&lt;/b&gt;</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="textLabel10">
                <property name="text">
                 <string>&lt;b&gt;h&lt;/b&gt;</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="textLabel10_4">
                <property name="text">
                 <string>&lt;b&gt;E&lt;/b&gt;</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>60</width>
                <height>16</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QGridLayout">
              <property name="margin">
               <number>0</number>
              </property>
              <property name="spacing">
               <number>15</number>
              </property>
              <item row="1" column="1">
               <widget class="QLineEdit" name="deltakInput">
                <property name="text">
                 <string>0.0</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_dk">
                <property name="text">
                 <string>&lt;b&gt;∆k&lt;/b&gt;</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="deltalInput">
                <property name="text">
                 <string>0.0</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_dh">
                <property name="text">
                 <string>&lt;b&gt;∆h&lt;/b&gt;</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="deltahInput">
                <property name="text">
                 <string>0.0</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLineEdit" name="deltaEInput">
                <property name="text">
                 <string>0.0</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_dl">
                <property name="text">
                 <string>&lt;b&gt;∆l&lt;/b&gt;</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_dnu">
                <property name="text">
                 <string>&lt;b&gt;∆E&lt;/b&gt;</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <spacer>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_dnu_2">
              <property name="text">
               <string>&lt;b&gt;∆q&lt;/b&gt;&amp;nbsp;&amp;nbsp;&amp;nbsp;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="deltaqInput">
              <property name="text">
               <string>0.0</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout">
            <property name="spacing">
             <number>15</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <spacer>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>86</width>
                <height>21</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="textLabel8_3">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>&lt;b&gt;Number of steps:&lt;/b&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="stepsInput">
              <property name="maximum">
               <number>100000</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>91</width>
                <height>26</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Preset</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QRadioButton" name="presetTime">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Time:</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="timeInput">
            <property name="maximum">
             <number>2000000000</number>
            </property>
            <property name="value">
             <number>120</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="presetMonitor">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Monitor:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="monitorInput">
            <property name="maximum">
             <number>2000000000</number>
            </property>
            <property name="value">
             <number>10000</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <layout class="QHBoxLayout">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <spacer>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>181</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="qscanCalc">
              <property name="text">
               <string>Calculate</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="tL_ttscan_End_3">
              <property name="text">
               <string>&lt;b&gt;Est. time:&lt;/b&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="qscanEstimation">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="cmdResult">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>141</width>
         <height>27</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="generateBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&amp;Generate Commands</string>
       </property>
       <property name="shortcut">
        <string>Alt+G</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearAllBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Clear &amp;All</string>
       </property>
       <property name="shortcut">
        <string>Alt+A</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="quitBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&amp;Quit</string>
       </property>
       <property name="shortcut">
        <string>Alt+Q</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="15" margin="15"/>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>scanStart</tabstop>
  <tabstop>scanStep</tabstop>
  <tabstop>scanNumsteps</tabstop>
  <tabstop>scanPreset</tabstop>
  <tabstop>scanCalc</tabstop>
  <tabstop>scanRange</tabstop>
  <tabstop>scanEstimation</tabstop>
  <tabstop>hInput</tabstop>
  <tabstop>kInput</tabstop>
  <tabstop>lInput</tabstop>
  <tabstop>EInput</tabstop>
  <tabstop>deltahInput</tabstop>
  <tabstop>deltakInput</tabstop>
  <tabstop>deltalInput</tabstop>
  <tabstop>deltaEInput</tabstop>
  <tabstop>stepsInput</tabstop>
  <tabstop>generateBtn</tabstop>
  <tabstop>clearAllBtn</tabstop>
  <tabstop>quitBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
